llvm/test/CodeGen/RISCV/rvv/fixed-vectors-fp.ll

```diff
@@ -3916,10 +3916,9 @@ define void @trunc_v6bf16(ptr %x) {
 ; CHECK-NEXT:    fmv.w.x fa5, a1
 ; CHECK-NEXT:    vsetivli zero, 8, e16, m1, ta, ma
 ; CHECK-NEXT:    vfwcvtbf16.f.f.v v10, v8
-; CHECK-NEXT:    vsetvli zero, zero, e32, m2, ta, ma
+; CHECK-NEXT:    vsetivli zero, 6, e32, m2, ta, ma
```

Comment:
This is from RISCVVLOptimizer being able to propagate VLs through mask users now